sound/soc/sof/pcm.c

```diff
@@ -285,9 +269,12 @@ static int sof_pcm_hw_params(struct snd_soc_component *component,
 		return ret;
 	}
 
-	ret = sof_pcm_dsp_params(spcm, substream, &ipc_params_reply);
-	if (ret < 0)
+	ret = snd_sof_ipc_pcm_params(sdev, substream, &ipc_params_reply);
+	if (ret < 0) {
+		dev_err(component->dev, "got wrong reply for PCM %d\n",
```

Comment:
maybe use __func__ to better localize the error, the message itself is somewhat non-descript.

Reply:
OK, I just moved the print and removed the `error:`